server/utils/assessmentUtils.test.ts

```diff
@@ -6,6 +6,9 @@ import {
   awaitingAssessmentTableRows,
   daysSinceInfoRequest,
   requestedFurtherInformationTableRows,
+  completedTableRows,
+  daysUntilDue,
+  formatDays,
```

Comment:
I think have a look at the imports here, and the exports in `assessmentUtils.ts` - looks like they've gotten out of sync with when they're need and should have been included across the last couple of commits